lib/perl/Genome/VariantReporting/Framework/Component/Report/MergeCompatible.pm

```diff
@@ -23,4 +23,10 @@ sub report_path {
         $self->class);
 }
 
+sub has_size {
+    my $self = shift;
+
+    return -s $self->report_path;
+}
```

Comment:
I pushed a commit to make has_size a transient attribute. I'm not 100% sure that the implementation caches successfully when the file doesn't have size since I believe that -s returns undef in this case. Suggestions are definitely welcome.